submissions/5mountains/tiny-js-world/index.js

```diff
@@ -0,0 +1,96 @@
+/* Refer to https://github.com/OleksiyRudenko/a-tiny-JS-world for the task details
+   Complete the below for code reviewers' convenience:
+
+   Code repository: _put repo URL here_
+   Web app: _put project's github pages URL here_
+   */
+
+// ======== OBJECTS DEFINITIONS ========
+// Define your objects here
+const man = {},
+   woman = {},
+   dog = {},
+   cat = {},
+   catWoman = {};
+
+man.species ='human';
+man.name = 'Tom';
+man.gender = 'male';
+man.legs = 2;
+man.hands = 2;
+man.paws = 0;
+man.saying = 'Hello Jenny!';
+man.friends = ['Tom', 'Jenny', 'Rex', 'Selina'];
+
+woman.species ='human';
+woman.name = 'Jenny';
+woman.gender = 'female';
+woman.legs = 2;
+woman.hands = 2;
+woman.paws = 0;
+woman.saying = 'Hello Tom!';
+woman.friends = ['Tom', 'Rex', 'Felix'];
+
+dog.species ='dog';
+dog.name = 'Rex';
+dog.gender = 'male';
+dog.legs = 0;
+dog.hands = 0;
+dog.paws = 4;
+dog.saying = 'woof-woof!';
+dog.friends = ['Tom', 'Jenny'];
+
+cat.species ='cat';
+cat.name = 'Felix';
+cat.gender = 'male';
+cat.legs = 0;
+cat.hands = 0;
+cat.paws = 4;
+cat.saying = 'meow!';
+cat.friends = ['Tom', 'Jenny', 'Selina'];
+
+catWoman.species = undefined;
+catWoman.name = 'Selina';
+catWoman.gender = 'female';
+catWoman.legs = undefined;
+catWoman.hands = undefined;
+catWoman.paws = undefined;
+catWoman.saying = cat.saying;
+catWoman.friends = ['Tom', 'Felix'];
```

Comment:
Having this definition for the catWoman the output looks as follows.
![image](https://user-images.githubusercontent.com/8686631/97734192-7eaa8400-1ae1-11eb-8931-74bc3043dc94.png)
Doesn't look like what you intended to have.

Reply:
Fixed! Please check the output now (: